.github/workflows/actions/patch_image_and_check_diff/action.yml

```diff
@@ -0,0 +1,196 @@
+# Composite action for patching image with release version
+name: Patch Image and Check Difference
+
+inputs:
+  repository:
+    description: "Repository where the workflow is running"
+    required: true
+  patch-image-arn:
+    description: "ARN of the release image that needs to be patched"
+    required: true
+  sample-app-namespace:
+    description: "Namespace where the sample app pods are located"
+    required: true
+
+outputs:
+  default-adot-image:
+    description: "adot image installed by latest version of eks-addon"
+    value: ${{ steps.default-adot-image.outputs.DEFAULT_ADOT_IMAGE }}
+  default-cw-agent-image:
+    description: "cw agent image installed by latest version of eks-addon"
+    value: ${{ steps.default-cw-agent-image.outputs.DEFAULT_CW_AGENT_IMAGE }}
+  default-cw-agent-operator-image:
+    description: "cw agent operator image installed by latest version of eks-addon"
+    value: ${{ steps.default-cw-agent-operator-image.outputs.DEFAULT_CW_AGENT_OPERATOR_IMAGE }}
+  latest-adot-image:
+    description: "adot image after patch"
+    value: ${{ steps.latest-adot-image.outputs.LATEST_ADOT_IMAGE }}
+  latest-cw-agent-image:
+    description: "cw-agent image after patch"
+    value: ${{ steps.latest-cw-agent-image.outputs.LATEST_CW_AGENT_IMAGE }}
+  latest-cw-agent-operator-image:
+    description: "cw-agent-operator image after patch"
+    value: ${{ steps.latest-cw-agent-operator-image.outputs.LATEST_CW_AGENT_OPERATOR_IMAGE }}
+  fluent-bit-image:
+    description: "fluent bit image installed by latest version of eks-addon"
+    value: ${{ steps.fluent-bit-image.outputs.FLUENT_BIT_IMAGE }}
```

Comment:
I was thinking we output all of these so that we can also double check if artifacts that weren't supposed to be changed remain the same. Additionally, during cw-agent-operator case we can change if other relevant artifacts were updated